protos/template/template.proto

```diff
@@ -51,4 +51,9 @@ message CreateResponse {
 
 message GetRequest {
   string id = 1;
+  string name = 2;
+}
+
+message FilterRequest {
+  string filter = 1;
 }
```

Comment:
I think it'd be great to have `FilterRequest` look like `GetRequest` so that we have one way to do filtering. This would also help to convey what field the filter is taking place on.

```proto
message FilterRequest {
  one_of filter_types {
    string name = 1;
  }
}
```